07.ls_object/file_detail.rb

```diff
@@ -40,7 +40,7 @@ def type
   end
 
   def mode
-    # ファイルモードを8進数に変換して末尾3桁（パーミッション）を取得する
+    # 8進数に変換したファイルモードの末尾3桁はパーミッションを表している
     @stat.mode.to_s(8)[-3..].each_char.map do |char|
```

Comment:
```suggestion
    permission = @stat.mode.to_s(8)[-3..]
    permission.each_char.map do |char|
```

みたいにすればコメントが不要になるかも（説明的変数の導入）

Reply:
コメントを削除して説明的変数を導入します。